src/Microsoft.VisualStudio.Editors/Microsoft.VisualStudio.Editors.pkgdef

```diff
@@ -1,4 +1,3 @@
-#include "versions.h"
```

Comment:
Would it be better to have a versions.h file with these constants defined esp if, based on our conversation yesterday, we might punt on auto-generation for a while? Or does that require some extra tools to run?